tests/smoke/test_mind.py

```diff
@@ -6,39 +6,38 @@
 import requests
 from reco_utils.dataset.mind import download_mind, extract_mind
 
+
 @pytest.mark.smoke
 @pytest.mark.parametrize("url, content_length, etag", 
-    [("https://recodatasets.z20.web.core.windows.net/newsrec/MINDdemo_train.zip",
-    "17372879", "0x8D82C63E386D09C"),
+    [
+    ("https://recodatasets.z20.web.core.windows.net/newsrec/MINDdemo_train.zip",
+    "17372879", '"0x8D8B8AD5B233930"'), # NOTE: the z20 blob returns the etag with ""
```

Comment:
@gramhagen this is super weird, it actually took me a while to debug, the etag of the new blob comes with "", whereas the other etags doesn't. Any idea why this could be happening?

Reply:
apparently this is by design: https://github.com/Azure/azure-storage-node/issues/603
must not be consistently implemented though...